python-sdk/nuimages/nuimages.py

```diff
@@ -276,9 +288,9 @@ def list_cameras(self) -> None:
                 channel_freqs[sensor['channel']] += 1
 
         # Print to stdout.
-        format_str = '{:7} {:6} {:24}'
+        format_str = '{:15} {:7} {:25}'
         print()
-        print(format_str.format('Cameras', 'Samples', 'Channel'))
+        print(format_str.format('Calibr. sensors', 'Samples', 'Channel'))
```

Comment:
Why do we need to have format_str here? Can't we directly print the strings?

Reply:
i guess to make it aligned in the table?

Reply:
They would not be aligned. It should look like a nice table.

Reply:
Ahh, got it.